src/server/pkg/hashtree/hashtree_bench_test.go

```diff
@@ -24,20 +24,23 @@ import (
 // Because of this, BenchmarkPutFile can be very slow for large 'cnt', often
 // much slower than BenchmarkMerge. Be sure to set -timeout 3h for 'cnt' == 100k
 //
-// Benchmarked times at rev. 27311193faf56f8e0e9a4e267ab6ea7abc1fe64e
+// Benchmarked times at rev. 6b8e9df38e42f624d2da0aaa785753e9e1d68c0d
 //  cnt |  time (s)
 // -----+-------------
-// 1k   |    0.000 s/op
-// 10k  |  145.139 s/op
-// 100k | 5101.328 s/op (1.4h)
+// 1k   | 0.007 s/op
+// 10k  | 0.088 s/op
+// 100k | 1.006 s/op (1.4h)
```

Comment:
No more `1.4h` right?  Also what's an `op` here?  Is that whatever that's inside the `b.N` block?